test/integration/param_passing.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#include <string>
+#include <gtest/gtest.h>
+#include "sdf/Filesystem.hh"
+#include "sdf/Root.hh"
+#include "test_config.h"
+
+void print_errors(sdf::Errors &_errors)
+{
+  for (sdf::Error e : _errors)
+    std::cout << e.Message() << std::endl;
+}
+
+/////////////////////////////////////////////////
+TEST(ParamPassingTest, ExperimentalParamsElement)
+{
+  const std::string modelRootPath =
+    sdf::filesystem::append(PROJECT_SOURCE_PATH, "test", "integration",
+                            "model") + "/";
```

Comment:
Would this cause issues in windows? I believe `sdf::filesystem::append` uses `\` in windows.

Reply:
5c167b32c5d79d57292702ebfb4114b39e4506cb